swo/README.md

```diff
@@ -0,0 +1,13 @@
+# Switchover (SWO)
+
+Switchover (SWO) is a feature that allows a live system to switch from one database to another safely and with little to no user impact.
+
+## Theory of Operation
+
+Switchover mode is initiated by starting GoAlert with an additional DB URL `--db-url-next`. The database referenced by `--db-url` is referred to as the "old" DB and the `--db-url-next` is the "new" DB.
```

Comment:
should this be updated to use `make start-swo`?

Reply:
That's just for development; this is the documentation for the operation of the feature itself (as it would be used in production). `make start-swo` is just setting `--db-url-next` under the covers.

That said, I'll add a `Development` section here noting that.

Reply:
sounds good! appreciate the clarification